New package: ChargedParticles v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: ChargedParticles
- Repository: https://github.com/Beforerr/ChargedParticles.jl
- Created by: @Beforerr
- Version: v0.1.0
- Commit: 6d4c36ffd88a781cf0ab6835e4a5ee8d6b57e3da
- Reviewed by: @Beforerr
- Reference: https://github.com/Beforerr/ChargedParticles.jl/issues/1#issuecomment-2477483608
- Description: A Julia package for representing charged particles
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598222e57e039d03f048d28002a7d1f694f18a37bbccc56acc310c83cea05eda49d22447e6828919be592a617aed36e2a73b3f8601ebb391dfd250fa01e0a645726ad56ce7b0f00d1e6bffb3294ed6ac7faec117e6ce129de49e0cec4d90c9fc1b6763e21355071e3b3e6c6dce5668949e7925fc9a303dfed0eed777e52a192fb60 -->

Beforerr commented:
I am wondering if there is anything I could do from my side to solve`API rate limit exceeded`.

JuliaRegistrator committed:
New package: ChargedParticles v0.1.0

UUID: e91dad46-2c06-47f2-8668-7ffa57fde2d9
Repo: https://github.com/Beforerr/ChargedParticles.jl.git
Tree: 54f25f04b87f47619394ae7a70999ca14b461bba

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
